tools/ci_build/github/linux/tvm/requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
```

Comment:
You also need to add the package here: `attrs`
If problems occur again, it will be necessary to check the need for the following packages:
* `tornado`
* `psutil`
* `xgboost`
* `cloudpickle`
* `pytest`